Volume/Models/ReadableContent.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ReadableContent.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 11/30/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+protocol ReadableContent: Hashable, Identifiable {
+    var id: String { get }
+    var publication: Publication { get }
+}
+
+extension ReadableContent {
```

Comment:
extensions should have a blank newline before and after the function based on our style guide